README.md

```diff
@@ -1 +1,519 @@
+
+* Create an account in New Relic
+* From the account dropdown, click API keys
+* Copy the license key which is also referenced in the UI as ingest - license
+
+## Environment Configuration
+
+We rely heavily on environment variables so that the correct log files are streamed to your observability dashboards. Ensure that you set the JF_PRODUCT_DATA_INTERNAL environment variable to the correct path for your product
+
+The environment variable JF_PRODUCT_DATA_INTERNAL must be defined to the correct location.
+
+Helm based installs will already have this defined based upon the underlying docker images.
+
+For non-k8s based installations below is a reference to the Docker image locations per product. Note these locations may be different based upon the installation location chosen.
+
+````text
+Artifactory: 
+export JF_PRODUCT_DATA_INTERNAL=/var/opt/jfrog/artifactory/
+````
+
+````text
+Xray:
+export JF_PRODUCT_DATA_INTERNAL=/var/opt/jfrog/xray/
+````
+
+````text
+Nginx:
+export JF_PRODUCT_DATA_INTERNAL=/var/opt/jfrog/nginx/
+````
+
+````text
+Mission Control:
+export JF_PRODUCT_DATA_INTERNAL=/var/opt/jfrog/mc/
+````
+
+````text
+Distribution:
+export JF_PRODUCT_DATA_INTERNAL=/var/opt/jfrog/distribution/
+````
+
+````text
+Pipelines:
+export JF_PRODUCT_DATA_INTERNAL=/opt/jfrog/pipelines/var/
+````
+
+## Fluentd Installation
+
+### OS / Virtual Machine
+
+Ensure you have access to the Internet from VM. Recommended install is through fluentd's native OS based package installs:
+
+| OS             | Package Manager        | Link                                                 |
+|----------------|------------------------|------------------------------------------------------|
+| CentOS/RHEL    | Linux - RPM (YUM)      | https://docs.fluentd.org/installation/install-by-rpm |
+| Debian/Ubuntu  | Linux - APT            | https://docs.fluentd.org/installation/install-by-deb |
+| MacOS/Darwin   | MacOS - DMG            | https://docs.fluentd.org/installation/install-by-dmg |
+| Windows        | Windows - MSI          | https://docs.fluentd.org/installation/install-by-msi |
+| Gem Install**	 | MacOS & Linux - Gem			 | https://docs.fluentd.org/installation/install-by-gem | 
+
+
+```text
+** For Gem based install, Ruby Interpreter has to be setup first, following is the recommended process to install Ruby
+
+1. Install Ruby Version Manager (RVM) as described in https://rvm.io/rvm/install#installation-explained, ensure to follow all the onscreen instructions provided to complete the rvm installation
+	* For installation across users a SUDO based install is recommended, the installation is as described in https://rvm.io/support/troubleshooting#sudo
+
+2. Once rvm installation is complete, verify the RVM installation executing the command 'rvm -v'
```

Comment:
Non-blocker: I prefer using backtick for cli commands to make them stand out more from regular text.